themes/openy_themes/openy_carnation/templates/page/page.html.twig

```diff
@@ -80,9 +80,9 @@
 
         {% if display_search %}
           <div class="search-form-wrapper col-12 border-top border-bottom">
-            <form method="get" action="{{ base_path }}/search/node">
+            <form method="get" action="{{ base_path }}/search">
               <i class="fa fa-search search-input-decoration" aria-hidden="true"></i>
-              <input type="search" name="keys" class="search-input" placeholder="Search">
+              <input type="search" name="q" class="search-input" placeholder="{{ 'Search'|t }}">
```

Comment:
We shouldn't break the back-compatibility. Leave 'keys' in carnation by default

Reply:
Make the key configurable. Make the search module configure this (theme) setting.

Reply:
Yes, it makes sense. @anprok we should use `name="keys"` by default and `name="q"` if Google Custom Search module is enabled.

Reply:
Replaced with template variable